swift/looker/Tests/lookerTests/methodsTests.swift

```diff
@@ -167,6 +167,15 @@ class methodsTests: XCTestCase {
         XCTAssertEqual("", result, result)
     }
 
+    func testGetAllUsersWithIds() {
+            let ids: [Int64] = [47, 48]
+            let users = try? sdk.ok(sdk.all_users(ids: ids))
+            XCTAssertEqual(users![0].first_name, "Ross")
+            XCTAssertEqual(users![0].last_name, "Bailey")
+            XCTAssertEqual(users![1].first_name, "Rob")
+            XCTAssertEqual(users![1].last_name, "Miller")
```

Comment:
Hah, well this might work for any generic instance 🤔

Reply:
Oh sure.

Reply:
Dang .. I thought you only had the test for Kotlin. Ok, I'll get to this tomorrow if I don't do it tonight.

Reply:
Ok, it's generic now. Sorry, Rob.